housing_cooperative_base/models/lease.py

```diff
@@ -162,26 +162,25 @@ def create_contract(self):
                     'recurring_invoices': True,
                     'lease_id': self.id,
                     'journal_id': self._default_journal().id,
-            })
-        )
+                }))
```

Comment:
mmh, matter of taste on that one: did you go through **black**?

Reply:
Didn't configure `black` yet. This probably happened unintentionally along the way.